libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -506,6 +512,14 @@ export class ModelOverview extends React.Component<
     this.setState({ showSplineChart: checked });
   };
 
+  private onBoxPlotStateUpdate = (
```

Comment:
should we add these changes under flight?

Reply:
Added flight newSdkEndpoints